charts/router/templates/router-clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+{{ if .Capabilities.APIVersions.Has (include "rbacApiVersion" .) }}
+kind: ClusterRole
+apiVersion: {{ template "rbacApiVersion" . }}
+metadata:
+  name: deis:deis-router
+  labels:
+    app: deis-router
+    heritage: deis
+rules:
+- apiGroups: ["extensions", "apps"]
+  resources: ["deployments"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["get", "list", "watch"]
```

Comment:
There's an issue open (https://github.com/deis/router/issues/274) that would require watch for services, but in the current implementation, watch isn't used (but list is). Could you please try removing it here and verify that the router still works? On principle, we should be giving the minimum set of permissions required.